src/main/kotlin/org/partiql/scribe/targets/spark/SparkAstToSql.kt

```diff
@@ -116,7 +117,12 @@ public open class SparkAstToSql(context: ScribeContext) : AstToSql(context) {
      * INT2 -> SMALLINT
      * INT4 -> INT
      * INT8 -> BIGINT
-     * DOUBLE_PRECISION -> DOUBLE
+     * DOUBLE PRECISION -> DOUBLE
+     * VARCHAR -> no params given default of 20 (https://spark.apache.org/docs/latest/api/java/org/apache/spark/sql/types/VarcharType.html#defaultSize())
+     * CHAR -> no params given default of 20 (https://spark.apache.org/docs/latest/api/java/org/apache/spark/sql/types/CharType.html#defaultSize())
+     * TIMESTAMP -> TIMESTAMP_NTZ; parameters removed
+     * TIMESTAMP WITH TIME ZONE -> TIMESTAMP; parameters removed
+     * TIME + TIME WITH TIME ZONE -> error since Spark does not have a TIME type
```

Comment:
self-review: regarding datetime types, SparkSQL does not support TIME/TIME WITH TIME ZONE.

TIMESTAMP in SparkSQL also corresponds to TIMESTAMP WITH TIME ZONE and TIMESTAMP_NTZ corresponds to TIMESTAMP WITHOUT TIME ZONE. Also, SparkSQL does not appear to allow for specifying parameters in a TIMESTAMP